app/services/idv/actions/verify_document_status_action.rb

```diff
@@ -0,0 +1,58 @@
+module Idv
+  module Actions
+    class VerifyDocumentStatusAction < Idv::Steps::VerifyBaseStep
+      def call
+        process_async_state(async_state)
+      end
+
+      private
+
+      def process_async_state(current_async_state)
+        render_json case current_async_state.status
+                    when :none
+                      { success: true, status: nil }
+                    when :in_progress
+                      { success: true, status: 'in_progress' }
+                    when :timed_out
+                      { success: false, status: nil, errors: ['timeout'] }
+                    when :done
+                      success = process_result(current_async_state.result)
+                      async_state_done if success
+                      { success: true, status: success ? 'success' : 'fail' }
```

Comment:
In the case of a failure, should we be expecting this response to be including the `errors` and `remaining_attempts` properties we were otherwise relying on from the image upload endpoint?

https://github.com/18F/identity-idp/blob/a57fc97/spec/controllers/idv/image_uploads_controller_spec.rb#L238-L242